src/shared/components/modals/LoginModal.tsx

```
@@ -1,15 +1,16 @@
+import { useKakaoLogin } from '@/shared/hooks';
 import { useModalStore } from '@/shared/store';
 
 import BaseModal from './BaseModal';
 
 const LoginModal = () => {
+  const { login } = useKakaoLogin();
   const closeModal = useModalStore(state => state.closeModal);
 
   const handleLogin = () => {
     closeModal();
+    login;
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)